### DISCUSSION TITLE
Dropdown 配制 Input 使用， 当 Input 设置 prefix 时， 点击 prefix 时 Dropdown 不会显示下拉

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#40639] [General]
Dropdown 配制 Input 使用， 当 Input 设置 prefix 时， 点击 prefix 时 Dropdown 不会显示下拉。

不知道是不是把事件给拦截了，禁止冒泡了
```
const items = [
  { value: '1', label: 'Item 1' },
  { value: '2', label: 'Item 2' },
  { value: '3', label: 'Item 3' }
]
<Dropdown
    menu={{
      items,
    }} trigger={['click']}
  >
    <Input prefix={<div style={{width:20, height:20,background:'red'}}></div>} />
</Dropdown>

<Dropdown
    menu={{
      items,
    }} trigger={['hover']}
  >
    <Input prefix={<div style={{width:20, height:20,background:'red'}}></div>} />
</Dropdown>
```

如果设置 trigger 为 click, 点击 prefix 的内容时， 不会显示下拉， 如果设置为 hover, 则移到输入框时就会显示，但是移到 prefix 内容上的时候，下拉就会隐藏了

演示地址： https://codesandbox.io/s/antd-dropdown-input-tv9gyh?file=/src/App.js

**Comment by Yuiai01** [comment]
是不是这样啊？ https://codesandbox.io/s/ji-ben-antd-5-1-7-forked-nc7t53?file=/demo.tsx:274-367

**Reply by ekuku1984** [reply]
不是这样， 我更新了代码， 之前贴的代码出了点问题

**Reply by Yuiai01** [reply]
> 不是这样， 我更新了代码， 之前贴的代码出了点问题

嗯嗯

**Comment by Wxh16144** [comment]
可以使用 [antd@5.x](https://u.ant.design/codesandbox-repro) 模板创建一个最小复现，方便大家帮助你排查问题。

**Reply by ekuku1984** [reply]
贴的代码变形了，我搞个 DEMO 上来
